src/utils/setLocalStorage.js

```diff
@@ -0,0 +1,19 @@
+export const setLocalStorage = (currentQuestion, quizData, key) => {
+  let answeredQuestions = localStorage.getItem('answeredQuestions');
```

Comment:
You can do 

`const answeredQuestions = localStorage.getItem('answeredQuestions') || [];`

To avoid re-initializing the variable in the line 12

Reply:
let answeredQuestions = localStorage.getItem('answeredQuestions') || [];I made it as like this but it gives an error.